docs/instrumentation-libraries.md

```diff
@@ -0,0 +1,121 @@
+# Support for OpenTelemetry Instrumentation Libraries
+
+OpenTelemetry instrumentation is implemented using a variety of techniques.
+Sometimes the instrumentation is built directly into the
+[instrumented library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumented-library),
+sometimes the instrumentation requires the addition of a separate
+[instrumentation library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library),
+and other times it is a hybrid of those approaches. This
+project aims to make using these instrumentation approaches easier and more
+automatic for our users. This requires us to both vet and dynamically enable a
+subset of the instrumentation that is available to the OpenTelemetry community,
+while still providing some flexibility to allow users to manually load and enable
+instrumentation that we do not support out of the box.
+
+## Libraries and our instrumentation support plans
+
+The libraries listed here contain our current thoughts about which libraries should
+include out of the box support, and which libraries we decided to not support. If
+you have a library that you would like to consider adding to this list, please
+submit an issue to request its inclusion.
+
+### Application Frameworks
+
+| Framework | Tracing Support | Metrics Support | Comments |
+| --- | :---: | :---: | --- |
+| [ASP.NET](https://docs.microsoft.com/en-us/aspnet/overview) | [Yes](./config.md#instrumented-traces-libraries-and-frameworks) | [Yes](./config.md#instrumented-metrics-libraries-and-frameworks) | |
```

Comment:
I suggest to put the hyperlinks to `config.md#instrumented-traces-libraries-and-frameworks` and `config.md#instrumented-metrics-libraries-and-frameworks` in the "header column" instead

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/931/commits/413ddefe5975877b141d044b6eff5329c002b1d8